pandas/core/window/rolling.py

```diff
@@ -1979,6 +1978,17 @@ def validate(self):
                 "closed only implemented for datetimelike and offset based windows"
             )
 
+    def _determine_window(self):
```

Comment:
pls return a value (and type it), rather than doing mutation; its not as obvious.

Reply:
Done.